.github/workflows/main.yml

```diff
@@ -148,8 +148,12 @@ jobs:
     if: github.event_name == 'push' && (contains(github.ref, 'master') || contains(github.ref, 'tags'))
     needs: [unit-bf, build-bf, integration]
     steps:
-      - name: log in
-        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

Comment:
Use the one on master
https://github.com/botfront/botfront/blob/master/.github/workflows/main.yml